dht.go

```diff
@@ -375,7 +375,7 @@ func (dht *IpfsDHT) Mode() ModeOpt {
 
 // fixLowPeersRoutine tries to get more peers into the routing table if we're below the threshold
 func (dht *IpfsDHT) fixLowPeersRoutine(proc goprocess.Process) {
-	timer := time.NewTimer(periodicBootstrapInterval)
+	timer := time.NewTicker(periodicBootstrapInterval)
```

Comment:
Thanks for the catch. Do you mind renaming timer -> ticker as well?

Reply:
I've changed the timer to ticker

Reply:
Thanks 😄